src/python/pants/backend/python/subsystems/python_repos.py

```diff
@@ -1,59 +1,7 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-import logging
-
-from pex.fetcher import Fetcher, PyPIFetcher
-from pex.http import RequestsContext, StreamFilelike, requests
-
 from pants.subsystem.subsystem import Subsystem
-from pants.util.memo import memoized_method
-
-
-logger = logging.getLogger(__name__)
-
-
-# TODO: These methods of RequestsContext are monkey-patched out to work around
-# https://github.com/pantsbuild/pex/issues/26: we should upstream a fix for this.
-_REQUESTS_TIMEOUTS = (15, 30)
-
-
-def _open_monkey(self, link):
```

Comment:
Just noting that the defaults in pex for timeouts (15s) and retries (5) now suffice to have removed this code. That said, opened https://github.com/pantsbuild/pex/issues/803 to expose configuration of these two settings in Pex which Pants may want to plumb at some point in the future.